src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
Yep, try printf_() in install.c.
Why do you need a return code or anything
like that? If you requested the script to install
ms-dos, it should either succeed or fail, but
it can't install, say, freedos instead. So I don't
see why you need the fancy return codes.
By the way I am really not sure if we need to
install any other doses than freedos to D:.
freedos can be installed system-wide, but for
other doses - why?